src/postgres/commands/createPostgresServer/IPostgresServerWizardContext.ts

```diff
@@ -8,7 +8,10 @@ import { IAzureDBWizardContext } from "../../../tree/IAzureDBWizardContext";
 
 export interface IPostgresServerWizardContext extends IAzureDBWizardContext {
 
-    adminUser?: string;
+    // username
+    shortUserName?: string;
```

Comment:
There's a standard way to write comments for properties and it looks like this:
```
/**
 * Username without server, i.e. "user1"
 */
shortUserName?: string;
```

One reason we do it this way is so that you can see the comment when you hover over the property or during autocomplete
<img width="647" alt="Screen Shot 2020-09-09 at 2 13 59 PM" src="https://user-images.githubusercontent.com/11282622/92655770-c1e93300-f2a6-11ea-9e5a-9a20e69e3930.png">